src/SourceLocator/SourceStubber/PhpStormStubsMap.php

```diff
@@ -0,0 +1,4910 @@
+<?php
```

Comment:
Up to the Roave team but might be worth adding a `Makefile` with a rule to auto-generate that file?

Reply:
Thinking again about it: if it's not too slow to generate maybe a Composer hook on install/update could be enough

Reply:
This may take a while, especially if the stubs reference each other somewhere (even in something as simple as `Exception`)

Reply:
Hmm, @theofidry may be right here, as we're unlikely going to be releasing faster than `jetbrains/phpstorm-stubs`. Adding a post-install hook is super-annoying though...

Reply:
The script should be used only when we upgrade `jetbrains/phpstorm-stubs` in https://github.com/Roave/BetterReflection/blob/master/composer.json#L8

So I don't understand why we should need composer hook.

Reply:
Ah, the stubs are locked to a specific version. I thought this was a version range selector